webapp/src/js/contest.coffee

```diff
@@ -367,10 +370,13 @@ class Run
 
 
 class Contest
-    constructor : (@contestTitle, @systemName, @systemVersion, problems=[], teams=[]) ->
+    constructor : (@contestTitle, @systemName, @systemVersion, @penaltyTime, problems=[], teams=[]) ->
+        if not @penaltyTime?
+            @penaltyTime = 20
```

Comment:
This can go wrong if penaltyTime is intentionally set to zero.

Reply:
And shall we move this parameter at the end?

Reply:
That if statement is not matter with zero issue, because coffee scripts will transpile as below:
```js
  Contest = (function() {
    function Contest(contestTitle, systemName, systemVersion, penaltyTime, problems, teams) {
      var p, pid, t, tid, _i, _j, _len, _len1;
      this.contestTitle = contestTitle;
      this.systemName = systemName;
      this.systemVersion = systemVersion;
      this.penaltyTime = penaltyTime;
      if (problems == null) {
        problems = [];
      }
      if (teams == null) {
        teams = [];
      }
      if (this.penaltyTime == null) {
        this.penaltyTime = 20;
      }
```

Reply:
I see. good point!